MetaMask/metamask-extension
# Preserve search query for "Swap from" and "Swap to" form fields, update content

## Explanation
This PR:
- Preserves search query for the `Swap from` and `Swap to` form fields
- Updates placeholder text on `Swap to` and `Swap from` search fields
- Updates copy on `slippage tolerance` tooltip

## Screenshots/Screencaps
A user typed `USD` into the `Swap to` field and clicked on `USDC`. When they click on the dropdown again, they will see this:
![image](https://user-images.githubusercontent.com/80175477/188888046-06c39a86-f00a-4e43-a1ab-c5bf7111574b.png)

Updated placeholder text on `Swap to` and `Swap from` search fields:
<img width="493" alt="image" src="https://user-images.githubusercontent.com/80175477/188888388-e45d11f3-66d1-44a1-8caf-036efc1b5007.png">

Updated copy on `slippage tolerance` tooltip:
<img width="529" alt="image" src="https://user-images.githubusercontent.com/80175477/188888254-be78846c-aa12-46d2-9133-aee157653a85.png">


## Manual Testing Steps
**Preserves search query for the `Swap from` and `Swap to` form fields**
- Type e.g. `USD` into the `Swap to` or `Swap from` fields and click on `USDC`. When you click on the dropdown again, your previous search query `USD` will be there

**Updates placeholder text on `Swap to` and `Swap from` search fields**
- Open Swaps and check the new placeholder text after clicking into the `Swap from` / `Swap to` form fields. Now it's `Search name or paste address` instead of `Search for a token`

**Updated copy on `slippage tolerance` tooltip**
- Check that the new content in the slippage tooltip contains `slippage tolerance` instead of `max slippage`
